post-result.py

```diff
@@ -10,7 +10,8 @@
 https://api.github.com/repos/RackHD/on-core/issues/${PullRequestId}/comments \
 http://rackhdci.lss.emc.com \
 http://rackhdci.lss.emc.com/job/on-core/851/ \
-'#851'
+'#851' \
+http://147.178.202.18/
```

Comment:
I will add a function to parse arguments which include "help" for each argument.